content/technotes/2020-02-26-tic-ghactions.Rmd

```diff
@@ -115,7 +118,7 @@ Users can change this behavior to their liking by changing the bash code that is
 ```
 
 The cache rebuilding happens in a scheduled CRON run daily at 3 am in the morning.
-Because tic always uploads the cache even if the build failed ([r-lib/actions](https://github.com/r-lib/actions) only uploads on a successful build), there will always be a cache available for builds which start during the day.
+Because tic always uploads the cache even if the build failed ([r-lib/actions](https://github.com/r-lib/actions) only uploads on a successful build), there will always be a cache available for builds which start during the day[^1].
```

Comment:
Hmm, tricky... Perhaps be a bit more explicit in the text what the problem is, and clarify that it's not tic's problem :) Just in case readers jump right in and are then confused that something doesn't work as expected.
Once again, this is assuming I've understood the context!
 
```suggestion
Because tic always uploads the cache even if the build failed ([r-lib/actions](https://github.com/r-lib/actions) only uploads on a successful build), there will always be a cache available for builds which start during the day. Unfortunately at the time of writing, GitHub actions do not support CRON events ("scheduled"). However, this is only a temporary state and will most likely change in the future (see [this issue](https://github.com/actions/cache/issues/63) for more information).
```

Reply:
I've clarified again - let me know if is more clear now.